atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1004,7 +1004,8 @@ void WebContents::InspectElement(int x, int y) {
   if (type_ == REMOTE)
     return;
 
-  OpenDevTools(nullptr);
+  if (!managed_web_contents()->GetDevToolsWebContents())
```

Comment:
> api::WebContents::InspectElement should just use the view of existing agent host instead of trying to create a new one, that way the desired output in the issue can be achieved.

@deepak1556 thanks for mentioning that, is this what you were thinking? This worked for me locally when calling `inspectElement` immediately after calling `openDevTools`.